src/TermUnification.js

```diff
@@ -14,14 +14,13 @@
  *  limitations under the License.
  */
 import {isBool, minBool, showBool} from "./Bools";
-import {boolUnify} from "./BoolUnification";
 import {applySubst, mergeSubst} from "./Substitution";
 import {isConstructor, mkConstructor, showTerm} from "./Terms";
 
 /**
  * Returns a substitution that unifies x and y (if it exists).
  */
-export function unifyTerm(t1, t2) {
+export function unifyTerm(t1, t2, unifyBool) {
```

Comment:
I guess unifyBool should really be called `unifyMethod` or similar, but OK. We can fix that later.

Reply:
or `unifyAlg` or whatever.

Reply:
OK, you know what, the name is fine.